content/_data/changelogs/weekly.yml

```diff
@@ -298,6 +298,7 @@
     - type: major rfe
       message: >
         <strong>Jenkins now requires Java 8 to run.</strong>
+         Note this applies to "agents" too (you can configure a custom <code>JavaPath</code> in their "Advanced" properties)!
```

Comment:
I have no idea what the part in parentheses refers to, and would be surprised if this is relevant to e.g. JNLP agents.